site/components/menu-collapsible/menu-collapsible.tsx

```diff
@@ -26,6 +27,21 @@ export const GitHubLaunch: React.FC = () => (
   </StyledAnchor>
 );
 
+export const DarkModeToggle: React.FC<{
```

Comment:
isn't it a bit better if that's called theme mode toggle
```suggestion
export const ThemeModeToggle: React.FC<{
```

Reply:
I'll reply to this in the next comment.